### ISSUE TITLE
Trouble running from AWS (Windows)

### ISSUE THREAD

**Issue by carljohanr** [featurecat/lizzie#275]
I'm having trouble getting this to run on an AWS P3 instance.

Progress so far:
- Started P3 instance with Windows
- Connected to instance through remote desktop
- Installed Firefox (IE seemed to have very strict security settings)
- Installed Lizzie, Java and Visual C++ 2015 Redistributable (issues with files not present)
- Opened Lizzie - I saw the board, but it reported that leelaz.exe had unexpectedly closed
- Tried both original network and Elf network - same results

Not sure how to get log information (tried PowerShell but no additional information). Grateful for any help, both for how to get it to work and how to streamline the initial process when launching the instance (since I don't want to keep it running).

**Comment by featurecat** [comment]
- try running Leela zero on its own to see if the executable works at all. "./leelaz -w network.gz". it should be able to start normally.
- try the cpu version which is more likely to work if graphics drivers are incorrectly configured. in that case, if the graphics are incorrectly configured, maybe the leela-zero repo can help you figure that out. they probably encountered this issue before. 
- (todo I really should add a debug mode, but I hope the above gives some clues) 

not sure about streamlining, I've done this before with different programs and it usually was just downloading Firefox. you could write a batch script that you paste into notepad that downloads all the files and opens each installation file. that would save you a lot of time. that's probably the best way.

**Comment by carljohanr** [comment]
It seems that if I run the leela command first it works. I did use a different instance type today (the so called Deep Learning one), but I still had the same issue when I opened Lizzie first. After I run the leelaz command and tuning, it worked fine. 

Any suggestions for how I can "standardize" the setup on a new AWS instance (lizzie + java 8 + elf network)? I can't afford to keep it running indefinitely...
